packages/commons/src/auth/jwt.ts

```diff
@@ -1,13 +1,30 @@
-import jwt, { JwtHeader, JwtPayload, SigningKeyCallback } from "jsonwebtoken";
+import jwt, { JwtHeader, JwtPayload, Secret } from "jsonwebtoken";
 import jwksClient from "jwks-rsa";
 import { invalidClaim, jwtDecodingError } from "pagopa-interop-models";
 import { JWTConfig, Logger } from "../index.js";
 import { AuthData, AuthToken, getAuthDataFromToken } from "./authData.js";
 
-export const decodeJwtToken = (jwtToken: string): JwtPayload | null => {
+export const decodeJwtToken = (
+  jwtToken: string,
+  logger?: Logger
```

Comment:
```suggestion
  logger: Logger
```

Reply:
handled in [a8f25fa](https://github.com/pagopa/interop-be-monorepo/pull/1002/commits/a8f25fa9566c0a64d239badba25795f6a4e04540)